script.js

```diff
@@ -44,10 +44,13 @@ $(document).ready(function () {
                     // songName: result[0].trackName,
                     // songURL: result[0].previewUrl
                 };
-                console.log("current log", result)
-                console.log("this is 100", result[0].artworkUrl100)
-                console.log("this is artist view url", result[0].artistViewUrl)
-                // $("#imageOutput").prepend(artistViewUrl);
+                // console.log("current log", result)
+                // console.log("this is 100", result[0].artworkUrl100)
+                // console.log("this is artist view url", result[0].artistViewUrl)
+                $("#artistInfo").attr("href", result[0].artistViewUrl);
+                $("#imageOutput").attr("src", result[0].artworkUrl100);
+                $("#imageOutput").attr("alt", result[0].artistName);
+                $("#imageOutput").attr("alt", result[0].artistName);
```

Comment:
Hey Casey did you mean to have lines 52 and 53 to be the same?

Reply:
Did not mean to have the same line twice.